app/javascript/components/pr-show.vue

```diff
@@ -1,135 +1,131 @@
+            {{ $t("message.prFeed.prAuthor") }}: {{ prAttribute(prCompleteInformation.owner_name ?
+              prCompleteInformation.owner_name : prCompleteInformation.owner_login) }}
+          </p>
+          <p>{{ $t("message.prFeed.prProject") }}: {{ prCompleteInformation.repository_name }}</p>
+          <p>{{ $t("message.prFeed.prTime") }}: {{ prTime(prCompleteInformation) }}</p>
+          <p>{{ $t("message.prFeed.prDate") }}: {{ prDate(prCompleteInformation) }}</p>
+          <p>{{ $t("message.prFeed.prDescription") }}: {{ prAttribute(prCompleteInformation.description) }}</p>
+          <div>
+            {{ $t("message.prFeed.prReviewers") }}:
+            <div
+              v-if="(prCompleteInformation.reviewers)"
+              style="flex: 2;"
+            >
+              <div
+                v-for="reviewer in prCompleteInformation.reviewers"
+                :key="reviewer.id"
+              >
+                <div v-if="(reviewer.github_user_name)">
+                  {{ reviewer.github_user_name }}
+                </div>
+                <div v-else>
+                  {{ reviewer.github_user_login }}
+                </div>
+              </div>
+            </div>
+            <div
+              v-else
+              style="flex: 2;"
+            >
+              {{ $t("message.prFeed.noName") }}
+            </div>
+          </div>
+          <p>{{ $t("message.prFeed.prCommits") }}: {{ prAttribute(prCompleteInformation.commits) }}</p>
+          <p>Likes: {{ prCompleteInformation.likes.total }}</p>
+          <a
+            target="_blank"
+            :href="prCompleteInformation.html_url"
+          > {{ $t("message.prFeed.prGoToGithub") }} </a>
+
+          <div v-if="!(prCompleteInformation.currentUserLike)">
+            <button
+              class="card-pr__button"
+              @click="toggleLike(prCompleteInformation)"
+            >
+              Like
+            </button>
+          </div>
+          <div v-else>
+            <button
+              class="card-pr__button"
+              @click="deleteLike(prCompleteInformation)"
+            >
+              Dislike
+            </button>
+          </div>
         </div>
       </div>
     </div>
   </div>
 </template>
 
 <script>
-
 import axios from 'axios';
 import moment from 'moment';
 
 export default {
   props: {
     pullRequest: {
+      type: Object,
+      required: true,
+    },
+    liked: {
       type: Array,
       required: true,
     },
-    likesGiven: {
+    reviewers: {
       type: Array,
       required: true,
     },
   },
 
   data() {
-    const pr = this.pullRequest;
-    const liked = this.likesGiven.find((like) => like.likeable_id === pr.id);
+    let prCompleteInformation = this.liked ?
+      { ...this.pullRequest, currentUserLike: this.liked } :
+      this.pullRequest;
+    prCompleteInformation = this.reviewers ?
+      { ...prCompleteInformation, reviewers: this.reviewers } :
+      prCompleteInformation;
```

Comment:
Si liked fuera un boolean podrías hacer esto así que es más simple y ordenado (si no hay reviewers, reviewers va a ser un array vacío).
```suggestion
    const prCompleteInformation =
      { ...this.pullRequest, currentUserLike: this.liked, reviewers: this.reviewers } 
```

Reply:
mucho mejor!